site-manage/pageorder/tool/src/java/org/sakaiproject/site/tool/helper/order/impl/SitePageEditHandler.java

```diff
@@ -508,6 +508,8 @@ public boolean disablePage(String pageId) throws SakaiException {
      * @throws IdUnusedException, PermissionException
      */
     public boolean enablePage(String pageId) throws SakaiException {
+
+        System.out.println("ENABLE PAGE: " + pageId);
```

Comment:
Nice.

Reply:
Well spotted. I wish I'd asked you for a review.

Reply:
@adrianfish it's just another example of bias, you were looking for one.